src/OpenTK/Platform/Windows/Bindings/HidProtocol.cs

```diff
@@ -28,7 +28,7 @@
 using System;
 using System.Runtime.InteropServices;
 using System.Security;
-using OpenTK.Platform.Common;
+using OpenTK.Input.Hid;
 
 namespace OpenTK.Platform.Windows
```

Comment:
Should this be moved to the NT assembly?

Reply:
Probably. I'll do it.

Reply:
After starting on this, I think this should be postponed. This will most likely require additional testing for almost all structs (it doesn't seem like everything is correct in its current state, so I don't just want to copy it 1:1), PLUS the documentation is very lacking (might be due to MSDN currently changing their documentation infrastructure).

Anyways, I'll just define this as "out of scope" for now, and will do this in another PR.

Reply:
Alright, sounds good.